packages/apollo-federation/src/composition/__tests__/compose.test.ts

```diff
@@ -994,9 +994,12 @@ describe('composeServices', () => {
         expect(errors).toHaveLength(0);
 
         const review = schema.getType('Review') as GraphQLObjectType;
-        expect(
-          review.getFields()['product'].federation.provides,
-        ).toMatchInlineSnapshot(`sku`);
+        expect(review.getFields()['product'].federation).toMatchInlineSnapshot(`
+          Object {
+            "provides": sku,
```

Comment:
Is there a reason `sku` is not in quotes? I'm not very familiar with the snapshot strategy.

Reply:
Ah I see. It's the snapshot serializer's description of a more complicated GraphQL field object.